torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3826,6 +3826,9 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 dtypes=[torch.cdouble], active_if=IS_WINDOWS),
                        SkipInfo('TestGradients', 'test_forward_mode_AD',
                                 dtypes=[torch.cdouble], active_if=IS_WINDOWS),
+                       # some test samples works for backward but not all
```

Comment:
A skip is OK for now. More correctly we should say that this op doesn't support backward for those dtypes, and we should revise the supported/unsupported tests as a collection so that a single sample input failure indicates a lack of support. Then we should file UX bugs for these cases (because operators either should or shouldn't support operations on certain dtypes, they shouldn't support them on only a subset of inputs).